nearlib/signing/json_account_info.js

```diff
@@ -0,0 +1,34 @@
+const KeyPair = require('./key_pair');
+
+/**
+ * Standard format for storing account info in json
+ */
+class JsonAccountInfo {
```

Comment:
this is very non-JS way to approach the problem. JS-way would be just have some helper function and call it on `json` vs wrapping an object around

Reply:
won't fix in this PL. I want to have more usages first to understand what the API needs to look like.

Reply:
What other usages are you thinking about? Why not inline this if you don't have more usages yet? Extract when there is actually more than one usage.